controllers/dockercluster_controller.go

```diff
@@ -39,26 +40,30 @@ type DockerClusterReconciler struct {
 func (r *DockerClusterReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	ctx := context.Background()
 	log := r.Log.WithValues("dockercluster", req.NamespacedName)
+	log.Info("Reconciling cluster")
 
 	dockerCluster := &infrastructurev1alpha2.DockerCluster{}
 	if err := r.Client.Get(ctx, req.NamespacedName, dockerCluster); err != nil {
 		if apierrors.IsNotFound(err) {
 			return ctrl.Result{}, nil
 		}
-		log.Error(err, "failed to get dockerMachine")
+		log.Error(err, "failed to get cluster")
 		return ctrl.Result{}, err
 	}
 
 	if dockerCluster.Status.Ready {
```

Comment:
Is this check needed anymore? I wouldn't expect reconcile to rely on a previously set status here.

Reply:
Yeah, I didn't think it was needed. Preserved it because I didn't :100: understand why it was there in the first place. I will remove it.

Reply:
Removed